yarn-project/simulator/src/avm/avm_simulator.test.ts

```diff
@@ -355,10 +360,10 @@ describe('AVM simulator', () => {
         expect(results.reverted).toBe(false);
 
         // Nullifier existence check should be in trace
-        const sideEffects = context.persistableState.flush();
-        expect(sideEffects.newNullifiers).toEqual([utxo]);
-        expect(sideEffects.nullifierChecks.length).toEqual(1);
-        expect(sideEffects.nullifierChecks[0].exists).toEqual(true);
+        const trace = context.persistableState.flush();
+        expect(trace.newNullifiers).toEqual([utxo]);
+        expect(trace.nullifierChecks.length).toEqual(1);
+        expect(trace.nullifierChecks[0].exists).toEqual(true);
```

Comment:
Maybe these two lines can be written as
`expect(trace.nullifierChecks).toEqual(expect.objectContaining({ exists: true }));`

https://jestjs.io/docs/expect#expectobjectcontainingobject

Reply:
I'm thinking I'll do a follow-up PR that changes a bunch of existing tests to use `objectContaining`